### PR TITLE
Fall 2019 web dev

### PR DESCRIPTION
**Pull request by jialuogao**

### CODE REVIEW

File: views/about.ejs [hunk 1]
```diff
@@ -9,179 +9,132 @@
...
+                </p>
+            </div>
+            <div class="col-sm-4">
+                <div class="img-center-holder">
+                    <img class="img-rounded home-pg-concept" src="/img/get-involved.png"><img>
+                </div>
+                <h3 class="img-center-holder">How to get involved</h3>
+                <p class="img-center-holder">
+                    The easiest way to get involved is to let us know your innovative ideas of new applications.
+                    If you feel confidant in your problem solving skills, you can also join us to help achieving the most
```
Comment (Vnorthway) [inline views/about.ejs]: Confident is spelled wrong on this line

File: views/user/email-confirmation.ejs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <% include ../template/head.ejs %>
+        <title>Verification required</title>
+    </head>
+    <body>
+        <% include ../template/nav.ejs %>
+
+        <div class="container body-layout ">
+            <div class="img-center-holder margin-dtop">
+                <img class="sponsor-img" src="img/icon.jpg" alt="Corgi">
+                <h2> A verification email has been sent to your email. Please verify your account, thank you! </h2>
```
Comment (mrr1vfe) [inline views/user/email-confirmation.ejs]: No need to add spaces in the front and at the end of this sentence

Comment (mrr1vfe) [reply views/user/email-confirmation.ejs]: Also, it would be good to change the color of icon.jpg from white to clear.

Comment (jialuogao) [reply views/user/email-confirmation.ejs]: Ok, the image can be edited (delete the background) using Photoshop or similar editing software and replace the old one. Do it if you can or Jiazhi (our board member) can take care of it

File: views/about.ejs (hunk 1 above)
Comment (jialuogao) [reply views/about.ejs]: fixed

**Pull request merged**